code/modules/vore/appearance/sprite_accessories_taur_vr.dm

```diff
@@ -45,8 +45,13 @@
 	buckle_lying = FALSE
 
 /mob/living/carbon/human/New()
-	..()
-	riding_datum = new /datum/riding/taur(src)
+	.=..()
+	spawn(2 SECONDS)
+	if(isTaurTail(tail_style))
```

Comment:
By making this check in New(),  people who gain taur tails after spawning (DNA or shapeshifting perhaps) won't get the verb, right?
Perhaps only those with extensive tauric experience can handle the power of riding.

Reply:
Yeah I suppose so. Gonna have to figure out a better spot to fit the things in. Seems that despite all the DNA and other setups during the spawn, the tail stuffs were taking their time before registering visible to isTaurTail()

Reply:
You might want to create a setTailType() helper, and in said helper, adjust their verbs if it's a taur tail or something.